gammapy/modeling/models/spatial.py

```diff
@@ -386,6 +387,40 @@ def plot_error(self, ax=None, **kwargs):
 
         return ax
 
+    def plot_extension_error(self, ax=None):
+        pass
+
+    def plot_error(self, ax, selection_optional="position", **kwargs):
```

Comment:
`selection_optional` is more suited when there is a set of default options which are executed, like in the `Estimators`. Maybe `which`?